security/certificates/service-serving-certificate.adoc

```diff
@@ -9,6 +9,9 @@ include::modules/customize-certificates-understanding-service-serving.adoc[level
 
 include::modules/customize-certificates-add-service-serving.adoc[leveloffset=+1]
 
+.Additional Resources
+You can use a configured service certificate to configure a secure route using reencrypt TLS termination. For more information, see xref:../../networking/routes/secured-routes.adoc#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes[Creating a re-encrypt route with a custom certificate].
```

Comment:
* The additional resources sections should always be a bulleted list, if you don't mind adding a bullet here
* There is both "configured" and "configure" in the first sentence. Any chance you could change either of those to a synonym?